src/doc/gui_manual/Preferences/File_Locations.rst

```diff
@@ -0,0 +1,297 @@
+* Written: When user :ref:`saves settings <How to Save Settings>`.
+* Read: On VisIt_ startup but this can be overridden by the ``-noconfig``
+  command-line :ref:`startup option <StartupOptions>`.
+* Format: ASCII `XML <https://en.wikipedia.org/wiki/XML>`_
+
+GUI Configuration File
+""""""""""""""""""""""
+* Location and file name: ``VUSER_HOME/guiconfig``
+* Purpose: Holds positions and sizes of various GUI windows. Also holds the list of
+  recently used paths to open databases.
+* Otherwise operates identically to ``VUSER_HOME/config``.
+
+:ref:`Host Profile <host_profiles>` Files
+"""""""""""""""""""""""""""""""""""""""""
+* Location and file name(s): ``VUSER_HOME/hosts/host_<site-name>_<resource-name>.xml``
+  where ``<site-name>`` is something like ``ornl``, ``llnl``, ``anl`` etc. and
+  ``<resource-name>`` is a machine name such as ``summit``, ``sierra``, ``theta``.
+* Purpose: Stores information on how to connect to and launch jobs on a specific
+  compute resource. In many cases, there are separate sets of host profile files
+  for all the compute resources at a commonly used site such as LLNL CZ or RZ,
+  ANL, ORNL, etc. Often sites will *post* VisIt_ host profile files in places for
+  users to easily find and *install* them. Installing them is just a matter of
+  copying them to ``VUSER_HOME``. In addition, updated profiles can be downloaded
+  and installed automatically by VisIt_ from the :ref:`Host Profiles <host_profiles>`
+  window.
+* Written: When user :ref:`saves settings <How to Save Settings>` or when user
+  hits the **Export Host** button from the :ref:`Host Profile <host_profiles>` window.
+* Read: On VisIt_ startup. All host profiles in ``VUSER_HOME/hosts/host*.xml`` are read
+  on VisIt_ startup but this can be overridden by ``-noconfig``. Users should be
+  aware of this behavior. If the user passes ``-noconfig`` for the purposes of
+  avoiding the loading of preferences, s/he will also be without any host profiles.
+* Format: ASCII `XML <https://en.wikipedia.org/wiki/XML>`_
+
+.. _file_locations_visitrc:
+
+VisIt Run Commands (rc) File
+""""""""""""""""""""""""""""
+* Location and file name: ``VUSER_HOME/visitrc``
+* Purpose: Holds Python code to be executed each time VisIt_ is launched.
+* Written: Whenever user hits the **Update Macros** button in the
+  :ref:`Command Window <Command_Line_Window>`.
+* Read: On VisIt_ startup of the CLI.
+* Format: Python source code. However, there is no ``.py`` file extension in the
+  file name.
+
+Command Window Tabs Script Files
+""""""""""""""""""""""""""""""""
+* Location and file name(s): ``VUSER_HOME/script<K>.py`` where ``K`` is an
+  integer in the range [1...8].
+* Purpose: Hold the python code associated with each tab in the
+  :ref:`Command Window <Command_Line_Window>`.
+* Written: When user :ref:`saves settings <How to Save Settings>`.
+* Read: On VisIt_ startup but this can be overridden by the ``-noconfig``
+* Format: Python source code.
```

Comment:
I think the following line is missing here:
command-line :ref:`startup option <StartupOptions>`.
or if the intent was to stop repeating the same information then removing "the" and adding a period.